docs/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only.md

```diff
@@ -0,0 +1,147 @@
+        /_/src/libraries/System.Linq/src/System/Linq/Select.cs(136,0): at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
+        ----- Inner Stack Trace -----
+        /_/src/libraries/System.Drawing.Common/src/System/Drawing/LibraryResolver.cs(31,0): at System.Drawing.LibraryResolver.EnsureRegistered()
+        /_/src/libraries/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs(65,0): at System.Drawing.SafeNativeMethods.Gdip.PlatformInitialize()
+        /_/src/libraries/System.Drawing.Common/src/System/Drawing/Gdiplus.cs(27,0): at System.Drawing.SafeNativeMethods.Gdip..cctor()
+```
+
+## Version introduced
+
+6.0 Preview 7
+
+## Type of breaking change
+
+This change breaks source compatibility.
+
+## Reason for change
+
+Because `System.Drawing.Common` was designed to be a thin wrapper over Windows technologies, its cross-platform implementation is subpar.
+
+`libgdiplus` is the main provider of the cross-platform implementation of `System.Drawing.Common` on the native side. `libgdiplus` is effectively a reimplementation of the parts of Windows that `System.Drawing.Common` depends on. That implementation makes `libgdiplus` a non-trivial component. It's around 30,000 lines of C code that's largely untested, and it lacks a lot of functionality. `libgdiplus` also has numerous external dependencies for image processing and text rendering, such as `cairo`, `pango`, and other native libraries. Those dependencies make maintaining and shipping the component even more challenging. Since the inclusion of the Mono cross-platform implementation, we have redirected numerous issues to `libgdiplus` that never got fixed. In comparison, other external dependencies we have taken, such as `icu` or `openssl`, are high-quality libraries. It's not viable to get `libgdiplus` to the point where its feature set and quality is on par with the rest of the .NET stack.
+
+From analysis of NuGet packages, we've observed that `System.Drawing.Common` is used cross-platform mostly for image manipulation, such as QR code generators and text rendering. We haven't noticed heavy graphics usage, as our cross-platform graphics support is incomplete. The usages we see of `System.Drawing.Common` in non-Windows environments are typically well supported with SkiaSharp and ImageSharp.
+
+`System.Drawing.Common` will continue to evolve only in the context of Windows Forms and GDI+.
+
+## Recommended action
+
+To use these APIs for cross-platform apps, migrate to one of the following libraries:
+
+- [ImageSharp](https://github.com/SixLabors/ImageSharp)
+- [SkiaSharp](https://github.com/mono/SkiaSharp)
+- [Microsoft.Maui.Graphics](https://github.com/dotnet/Microsoft.Maui.Graphics)
+
+Alternatively, you can enable Unix support by setting the `System.Drawing.EnableUnixSupport` run-time configuration switch to `true` in the *runtimeconfig.json* file:
```

Comment:
Should `run-time` be `runtime`?

Reply:
I'm torn about this. Are we configuring the .NET runtime, or are we configuring how the app behaves at run time?

Reply:
how it behaves at run time, however the feature is called `runtime switch` because are read by the host and then the `runtime` sets them and we fetch them from the runtime.